app/views/onestring.php

```diff
@@ -23,13 +23,17 @@
     <tr>
         <th>Locale</th>
         <th>Translation</th>
+        <th>&nbsp;</th>
     </tr>
     <?php
+    $reference_locale = Project::getReferenceLocale($repo);
```

Comment:
ok, I think it wouldn't be any problem with phpn 5.5 because it has a built-in opt-code cache but indeed we use 5.4 and we don't have one (yet)